server/chat-commands/room-settings.ts

```diff
@@ -741,9 +741,9 @@ export const commands: ChatCommands = {
 			return this.errorReply(`Your group chat name is too similar to existing chat room '${title}'.`);
 		}
 		// Room IDs for groupchats are groupchat-TITLEID
-		let titleid = toID(title);
+		const titleid = toID(title);
 		if (!titleid) {
-			titleid = `${Math.floor(Math.random() * 100000000)}` as ID;
+			title = `${Math.floor(Math.random() * 100000000)}` as ID;
```

Comment:
This change seems wrong to me. If the title is "こんにちは", the title should be unchanged and a random-number title should be generated.

```suggestion
			titleid = `${Math.floor(Math.random() * 100000000)}` as ID;
			if (!title) title = titleid;
```

Reply:
i didn't realize that was intended so i'll undo that then